### DISCUSSION TITLE
Hold right click scrolling in song menu like in classic osu

### DISCUSSION THREAD

**Discussion by falsestatement** [ppy/osu#26034] [Ideas] [answered]
Just as the title suggests, add back the ability to quickly navigate song menu by holding down right click

**Comment by Flamiii** [comment, marked as answer]
This is in the game already. It's the setting "Right mouse drag to absolute scroll".

**Reply by TheEZIC** [reply]
Thx you! I don't know about that as well. Maybe this option should be enabled by default?

**Reply by Taoshix** [reply]
during the setup wizard you can choose to use classic default settings where that is one of the settings that gets enabled

**Reply by TheEZIC** [reply]
Ok thx you too. I've never payed attention on that.
